src/runtime/graph/graph_runtime.cc

```diff
@@ -55,6 +55,17 @@ class GraphRuntime : public ModuleNode {
       if (op_execs_[i]) op_execs_[i]();
     }
   }
+  /*!
+   * Run the graph up to the given node.
+   */
+  void RunPartial(int index) {
+    // setup the array and requirements.
+
+    for (size_t i = 0; i < op_execs_.size(); ++i) {
+      if (static_cast<int>i == index) return;
```

Comment:
The code do not build, merge this function with DebugGetOutput

Reply:
Done.